README

```diff
@@ -1,8 +1,51 @@
-Welcome and thanks for downloading uHub, a high performance ADC p2p hub.
+# Uhub
 
-For the official documentation, bugs and other information, please visit:
-https://www.uhub.org/
+Uhub is a high performance peer-to-peer hub for the ADC network. Its low memory footprint allows it to handle several thousand users on high-end servers, or a small private hub on embedded hardware.  
```

Comment:
I think [ADC](https://en.wikipedia.org/wiki/Advanced_Direct_Connect) should be link here and maybe full name ([Advanced Direct Connect](https://en.wikipedia.org/wiki/Advanced_Direct_Connect)) would be more preferable.